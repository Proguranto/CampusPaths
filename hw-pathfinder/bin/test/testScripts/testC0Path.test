# Find a path with Dijkstra's algorithm when
# the graph is cyclic and there is no path
# from start to dest node.

CreateGraph graph2

AddNode graph2 S
AddNode graph2 1
AddNode graph2 2
AddNode graph2 D

AddEdge graph2 S 1 1.0
AddEdge graph2 1 2 1.0
AddEdge graph2 2 S 1.0

FindPath graph2 S D
