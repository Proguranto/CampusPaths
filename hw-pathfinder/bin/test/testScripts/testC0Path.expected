# Find a path with Dijkstra's algorithm when
# the graph is cyclic and there is no path
# from start to dest node.

created graph graph2

added node S to graph2
added node 1 to graph2
added node 2 to graph2
added node D to graph2

added edge 1.000 from S to 1 in graph2
added edge 1.000 from 1 to 2 in graph2
added edge 1.000 from 2 to S in graph2

path from S to D:
no path found
